frontend/src/pages/Seller/Auth/Login.tsx

```diff
@@ -1,9 +1,70 @@
-import * as React from "react"
-const Login: React.FC<{}> = (props) => {
+// Login.tsx
+
+import * as React from "react";
+import { Input, Button, Box, Flex } from '@chakra-ui/react';
+import './Login.css'; // Import the new CSS file
+
+interface LoginProps {}
+
+const Login: React.FC<LoginProps> = () => {
+    const [username, setUsername] = React.useState<string>('');
+    const [password, setPassword] = React.useState<string>('');
+
+    const handleLogin = () => {
+
+        console.log('Logging in with:', { username, password });
+    };
+
     return (
-        <div>
-            <h1>Login</h1>
-        </div>
+        <Flex className="content">
+
+            <Box className="gradient-bg" />
+
+
+            <Box className="login-container">
+                <h1 className="login-header">Seller Login</h1>
+                <p className="login-text">Continue to Hersey Spirit wear Dashboard</p>
+
+
+                <Input
+                    placeholder="Username"
+                    value={username}
+                    onChange={(e) => setUsername(e.target.value)}
+                    mb={4}
+                />
+
+
+                <Input
+                    type="password"
+                    placeholder="Password"
+                    value={password}
+                    onChange={(e) => setPassword(e.target.value)}
+                    mb={6}
+                />
+
+
+                <Button
+                    colorScheme="blue"
+                    onClick={handleLogin}
+                    w="100%"
+                >
+                    Log In
+                </Button>
+
+
+                <Button
```

Comment:
(Repeat) Add icons to the buttons as per the designs. I added a resource for this in the last PR review.